subsys/bluetooth/controller/include/ll_feat.h

```diff
@@ -189,10 +189,20 @@
 #define LL_FEAT_BIT_CIS_PERIPHERAL 0
 #endif /* !CONFIG_BT_CTLR_PERIPHERAL_ISO */
 
+#if defined(CONFIG_BT_CTLR_ADV_ISO) || defined(CONFIG_BT_CTLR_CONN_ISO)
+#if defined(CONFIG_BT_ISO_TX_MTU)
+#define BT_CTLR_ISO_TX_BUFFER_SIZE MIN((CONFIG_BT_ISO_TX_MTU + 8U), \
```

Comment:
Consider using the macro `BT_HCI_ISO_TS_DATA_HDR_SIZE` for places where you currently use `8` (or `BT_HCI_ISO_TS_DATA_HDR_SIZE + 1` instead of `9`).

Reply:
`struct bt_hci_iso_data_hdr`  naming is misleading, adds to the confusion!

Reply:
Not fully disagree or agree with that :D But it's certainly better than magic numbers. 

But yeah, I see how the the data header can be confused with the packet header.